components/financial_assistance/app/views/financial_assistance/incomes/other.html.erb

```diff
@@ -61,78 +61,78 @@
     </div>
     <% FinancialAssistance::Income::OTHER_INCOME_TYPE_KIND.each do |other_income_kind, label| %>
```

Comment:
the note on line 60 stays opaque, but the label of Note does not. Very minor, but looks off. it is a pertinent note, so i would adjust so the note label also stays bold